From 62d2503ac77f9f3ca5e660698000a1eb2c2bddd9 Mon Sep 17 00:00:00 2001
From: Gustaw Lewandowski <gustaw.lewandowski@intel.com>
Date: Mon, 17 Sep 2018 11:27:02 +0200
Subject: [PATCH 2151/2367] REVERTME: Increase tdf8532 timeout and number of
 retries for audio ROM

This is WA for delays in ROM init and ACK timeout for tdf8532 codec.
New values at this patch are set to very high so this patch should be
reverted after RC will be know or new reasonable values be established.

Tracked-On: https://jira01.devtools.intel.com/browse/OAM-67068
Change-Id: I4e8145a8711ef9ff737a80028f9e22d9a991f4b4
Signed-off-by: Gustaw Lewandowski <gustaw.lewandowski@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/31883
Reviewed-by: Rojewski, Cezary <cezary.rojewski@intel.com>
Tested-by: Rojewski, Cezary <cezary.rojewski@intel.com>
---
 sound/soc/codecs/tdf8532.h        | 2 +-
 sound/soc/intel/skylake/bxt-sst.c | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/codecs/tdf8532.h b/sound/soc/codecs/tdf8532.h
index 6e3f2c1..39353a0 100644
--- a/sound/soc/codecs/tdf8532.h
+++ b/sound/soc/codecs/tdf8532.h
@@ -16,7 +16,7 @@
 #ifndef __TDF8532_H_
 #define __TDF8532_H_
 
-#define ACK_TIMEOUT 300
+#define ACK_TIMEOUT 10000
 
 #define CHNL_MAX 5
 
diff --git a/sound/soc/intel/skylake/bxt-sst.c b/sound/soc/intel/skylake/bxt-sst.c
index e6b6a17..97a3210 100644
--- a/sound/soc/intel/skylake/bxt-sst.c
+++ b/sound/soc/intel/skylake/bxt-sst.c
@@ -59,7 +59,7 @@
 /* Delay before scheduling D0i3 entry */
 #define BXT_D0I3_DELAY 5000
 
-#define BXT_FW_INIT_RETRY 3
+#define BXT_FW_INIT_RETRY 100
 
 #define GET_SSP_BASE(N)	(N > 4 ? 0x2000 : 0x4000)
 
-- 
2.7.4

